

<!doctype html>
<html lang="en" class="no-js">
  <head>
    

<meta charset="utf-8">



<!-- begin SEO -->









<title>Publications - Dzakwan Falih</title>







<meta property="og:locale" content="en-US">
<meta property="og:site_name" content="Dzakwan Falih">
<meta property="og:title" content="Publications">


  <link rel="canonical" href="http://localhost:4000/publications/">
  <meta property="og:url" content="http://localhost:4000/publications/">





  <meta name="twitter:site" content="@dzakwanfalihh">
  <meta name="twitter:title" content="Publications">
  <meta name="twitter:description" content="Dzakwan Falih">
  <meta name="twitter:url" content="http://localhost:4000/publications/">

  
    <meta name="twitter:card" content="summary">
    
  

  



  

  












  <script type="application/ld+json">
    {
      "@context" : "http://schema.org",
      "@type" : "Person",
      "name" : "Dzakwan Falih",
      "url" : "http://localhost:4000",
      "sameAs" : null
    }
  </script>






<!-- end SEO -->


<link href="http://localhost:4000/feed.xml" type="application/atom+xml" rel="alternate" title="Dzakwan Falih Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- For all browsers -->
<link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">

<meta http-equiv="cleartype" content="on">
    

<!-- start custom head snippets -->

<link rel="apple-touch-icon" sizes="57x57" href="http://localhost:4000/images/apple-touch-icon-57x57.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="60x60" href="http://localhost:4000/images/apple-touch-icon-60x60.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="72x72" href="http://localhost:4000/images/apple-touch-icon-72x72.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="76x76" href="http://localhost:4000/images/apple-touch-icon-76x76.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="114x114" href="http://localhost:4000/images/apple-touch-icon-114x114.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="120x120" href="http://localhost:4000/images/apple-touch-icon-120x120.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="144x144" href="http://localhost:4000/images/apple-touch-icon-144x144.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="152x152" href="http://localhost:4000/images/apple-touch-icon-152x152.png?v=M44lzPylqQ">
<link rel="apple-touch-icon" sizes="180x180" href="http://localhost:4000/images/apple-touch-icon-180x180.png?v=M44lzPylqQ">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-32x32.png?v=M44lzPylqQ" sizes="32x32">
<link rel="icon" type="image/png" href="http://localhost:4000/images/android-chrome-192x192.png?v=M44lzPylqQ" sizes="192x192">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-96x96.png?v=M44lzPylqQ" sizes="96x96">
<link rel="icon" type="image/png" href="http://localhost:4000/images/favicon-16x16.png?v=M44lzPylqQ" sizes="16x16">
<link rel="manifest" href="http://localhost:4000/images/manifest.json?v=M44lzPylqQ">
<link rel="mask-icon" href="http://localhost:4000/images/safari-pinned-tab.svg?v=M44lzPylqQ" color="#000000">
<link rel="shortcut icon" href="/images/favicon.ico?v=M44lzPylqQ">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-TileImage" content="http://localhost:4000/images/mstile-144x144.png?v=M44lzPylqQ">
<meta name="msapplication-config" content="http://localhost:4000/images/browserconfig.xml?v=M44lzPylqQ">
<meta name="theme-color" content="#ffffff">
<link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.css"/>
<!-- end custom head snippets -->

  </head>

  <body>

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->
    

<div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <button><div class="navicon"></div></button>
        <ul class="visible-links">
          <li class="masthead__menu-item masthead__menu-item--lg"><a href="http://localhost:4000/">Dzakwan Falih</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/blog/">Blog</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/publications/">Publications</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/cv/">CV</a></li>
          
            
            <li class="masthead__menu-item"><a href="http://localhost:4000/travel/">Road trip</a></li>
          
        </ul>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    





<div id="main" role="main">
  


  <div class="sidebar sticky">
  



<div itemscope itemtype="http://schema.org/Person">

  <div class="author__avatar">
    
    	<img src="http://localhost:4000/images/gue.jpeg" class="author__avatar" alt="Dzakwan Falih">
    
  </div>

  <div class="author__content">
    <h3 class="author__name"><a href="/">Dzakwan Falih</a></h3>
    <p class="author__bio">reformed cyber criminal @ cyber space</p>
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      
        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> Cyber Space, Internet</li>
      
      
      
      
        <li><a href="https://keybase.io/dzakwanfalih"><i class="fa fa-fw fa-key" aria-hidden="true"></i> Keybase</a></li>
      
      
        <li><a href="https://twitter.com/dzakwanfalihh"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
      
      
      
      
        <li><a href="https://github.com/dzakwanfalih"><i class="fa fa-fw fa-github" aria-hidden="true"></i> Github</a></li>
      
      
      
        <li><a href="https://instagram.com/dzakwanfalih_2.0"><i class="fa fa-fw fa-instagram" aria-hidden="true"></i> Instagram</a></li>
      
      
      
      
      
      
      
      
      
      
      
      
      
      
      
      
        <li><a href="https://scholar.google.com/citations?hl=en&user=ZFuwjiIAAAAJ"><i class="ai ai-google-scholar-square ai-fw"></i> Google Scholar</a></li>
      
      
      
      
      
        <li><a href="mailto:dzakwanfaliht@hotmail.com"><i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> Email</a></li>
      

    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="http://schema.org/CreativeWork">
    <meta itemprop="headline" content="Publications">
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 class="page__title" itemprop="headline">Publications
</h1>
          
        
        
        
        
             
        
    
        </header>
      

      <section class="page__content" itemprop="text">
        
<h2 id="academic-publications">Academic publications</h2>

<p><strong>Claire Duvallet</strong>, Bryan D Hayes, Timothy B Erickson, Peter R Chai, Mariana Matus. <br />
“Mapping community opioid exposure through wastewater-based epidemiology as a
means to engage pharmacies in harm reduction efforts.” <br />
<em>Preventing Chronic Disease</em> (2020). doi: <a href="http://dx.doi.org/10.5888/pcd17.200053">10.5888/pcd17.200053</a></p>

<p>Fuqing Wu, Amy Xiao, Jianbo Zhang, … <strong>Claire Duvallet</strong> … Eric J Alm.  <br />
“SARS-CoV-2 titers in wastewater foreshadow dynamics and clinical presentation of new COVID-19 cases.”  <br />
<em>medRxiv</em> (2020). doi: <a href="https://www.medrxiv.org/content/10.1101/2020.06.15.20117747v2">10.1101/2020.06.15.20117747v2</a></p>

<p>Fuqing Wu, Amy Xiao, Jianbo Zhang, Xiaoqiong Gu, Wei Lin Lee, Kathryn Kauffman, William Hanage, Mariana Matus, Newsha Ghaeli, Noriko Endo, <strong>Claire Duvallet</strong>, Katya Moniz, Timothy Erickson, Peter Chai, Janelle Thompson, Eric Alm.  <br />
“SARS-CoV-2 titers in wastewater are higher than expected from clinically confirmed cases.” <br />
<em>mSystems</em> (2020). doi: <a href="https://msystems.asm.org/content/5/4/e00614-20">10.1128/mSystems.00614-20</a></p>

<p><strong>Claire Duvallet</strong>.   <br />
“Data detectives, self-love, and humility: a research parasite’s perspective.”  <br />
<em>GigaScience</em> (2020). doi: <a href="https://doi.org/10.1093/gigascience/giz148">10.1093/gigascience/giz148</a>.</p>

<p>Noriko Endo, Newsha Ghaeli, <strong>Claire Duvallet</strong>, Katelyn Foppe, Timothy B Erickson, Mariana Matus, Peter R Chai. <br />
“Rapid Assessment of Opioid Exposure and Treatment in Cities Through Robotic Collection and Chemical Analysis of Wastewater.”   <br />
<em>Journal of Medical Toxicology</em> (2020) doi: <a href="https://doi.org/10.1007/s13181-019-00756-5">10.1007/s13181-019-00756-5</a>.</p>

<p>Ethan D Evans, <strong>Claire Duvallet</strong>, Nathaniel D Chu, Michael K Oberst, Michael A Murphy, Isaac Rockafellow, David Sontag, Eric J Alm. <br />
“Predicting human health from biofluid-based metabolomics using machine learning.” <br />
<em>medRxiv</em> (2020) doi: <a href="https://doi.org/10.1101/2020.01.29.20019471">10.1101/2020.01.29.20019471</a>.</p>

<p>Mariana Matus, <strong>Claire Duvallet</strong>, Newsha Ghaeli, Melissa Kido Soule, Krista Longnecker, Ilana Brito, Carlo Ratti, Elizabeth B. Kujawinski, Eric Alm. <br />
“Untargeted detection of human health and activity markers in residential wastewater through microbiome sequencing and metabolomics.”  <br />
<em>bioRxiv</em> (2019) doi: <a href="https://doi.org/10.1101/728022">10.1101/728022</a>.</p>

<p><strong>Claire Duvallet</strong>, Caroline Zellmer, Pratik Panchal, Shrish Budree, Madji Osman, and Eric Alm.  <br />
“Framework for rational donor selection in fecal microbiota transplant clinical trials.”  <br />
<em>PLoS ONE</em> (2019) doi: <a href="https://doi.org/10.1371/journal.pone.0222881">10.1371/journal.pone.0222881</a>.   <br />
Associated github: <a href="https://github.com/cduvallet/donor-selection/">cduvallet/donor-selection</a></p>

<p>Evan Bolyen, Jai Ram Rideout, Matthew R. Dillon, Nicholas A. Bokulich … <strong>Claire Duvallet</strong> … Rob Knight, Greg Caporaso.   <br />
“Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2.”    <br />
<em>Nature Biotechnology</em> (2019). doi: <a href="https://doi.org/10.1038/s41587-019-0209-9">10.1038/s41587-019-0209-9</a>.</p>

<p>Chengzhen Dai, <strong>Claire Duvallet</strong>, An Ni Zhang, Mariana Matus, Newsha Ghaeli, Shinkyu Park, Noriko Endo, Siavash Isazadeh, Kazi Jamil, Carlo Ratti, and Eric Alm.  <br />
“Multi-site sampling and risk prioritization reveals the public health relevance of antibiotic resistance genes found in sewage environments.”    <br />
<em>bioRxiv</em> (2019). doi: <a href="https://doi.org/10.1101/562496">10.1101/562496</a>.</p>

<p><strong>Claire Duvallet</strong>, Kara Larson, Scott Snapper, Sonia Iosim, Ann Lee, Katherine Freer, Kara May, Eric Alm, and Rachel Rosen.  <br />
“Aerodigestive sampling reveals altered microbial exchange between lung, oropharyngeal, and gastric microbiomes in children with impaired swallow function.”   <br />
<em>PLOS ONE</em> (2019). doi: <a href="https://doi.org/10.1371/journal.pone.0216453">10.1371/journal.pone.0216453</a>.  <br />
Associated github: <a href="https://github.com/cduvallet/aspiration-analysis-public">cduvallet/aspiration-analysis-public</a></p>

<p>Keegan Korthauer*, Patrick K Kimes*, <strong>Claire Duvallet</strong>^, Alejandro Reyes^, Ayshwarya Subramanian^, Mingxiang Teng, Chinmay Shukla, Eric J Alm, and Stephanie C Hicks.   <br />
“A practical guide to methods controlling false discoveries in computational biology.”   <br />
<em>Genome Biology</em> (2019). doi: <a href="https://doi.org/10.1186/s13059-019-1716-1">10.1186/s13059-019-1716-1</a>.  <br />
Associated github: <a href="https://github.com/pkimes/benchmark-fdr">pkimes/benchmark-fdr</a></p>

<p>Thomas Gurry, HST Microbiome Consortium, Sean M Gibbons, Sean M Kearney, Ashwin Ananthakrishnan, Xiaofang Jiang, <strong>Claire Duvallet</strong>, Zain Kassam, and Eric J Alm.    <br />
“Predictability and persistence of prebiotic dietary supplementation in a healthy human cohort.”     <br />
<em>Scientific Reports</em> (2018) 8(1): 12699. doi: <a href="https://doi.org/10.1038/s41598-018-30783-1">10.1038/s41598-018-30783-1</a></p>

<p>Sean M. Gibbons, <strong>Claire Duvallet</strong>, and Eric J. Alm.    <br />
“Correcting for batch effects in case-control microbiome studies.”     <br />
<em>PLoS Computational Biology</em> (2018) 14(4): e1006102. doi: <a href="https://doi.org/10.1371/journal.pcbi.1006102">10.1371/journal.pone.0176335</a></p>

<p><strong>Claire Duvallet</strong>. “Meta-analysis generates and prioritizes hypotheses for translational microbiome research.”      <br />
<em>Microbial Biotechnology</em> (2018). doi: <a href="https://doi.org/10.1111/1751-7915.13047">10.1111/1751-7915.13047</a></p>

<p><strong>Claire Duvallet</strong>, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm.     <br />
“Meta-analysis of gut microbiome studies identifies disease-specific and shared responses.”      <br />
<em>Nature Communications</em> (2017). doi: <a href="https://doi.org/10.1038/s41467-017-01973-8">10.1038/s41467-017-01973-8</a>.     <br />
Associated github: <a href="https://github.com/cduvallet/microbiomeHD">cduvallet/microbiomeHD</a>   <br />
Associated data: <a href="https://doi.org/10.5281/zenodo.797943">zenodo record 840333</a>.</p>

<p>Scott W. Olesen, <strong>Claire Duvallet</strong>, and Eric J. Alm.     <br />
“dbOTU3: A new implementation of distribution-based OTU calling.”    <br />
<em>PloS ONE</em> 12, no. 5 (2017): e0176335. doi: <a href="https://doi.org/10.1371/journal.pone.0176335">10.1371/journal.pone.0176335</a></p>

<h2 id="non-peer-reviewed-academic-content">Non-peer reviewed academic content</h2>

<p><em>[Tutorial]</em> <strong>Claire Duvallet</strong>. “<a href="https://dev.qiime2.org/latest/tutorials/updating-plugin/">Updating your qiime2 plugin</a>.” <em>QIIME 2 developer docs.</em> (2019).</p>

<p><em>[Blog post]</em> <strong>Claire Duvallet</strong>. “<a href="https://go.nature.com/30rx4VZ">Scientific discovery from a clinical study: surprises from the lung and stomach microbiomes.</a>” <em>Nature Microbiology Community Forum.</em> May 2019.</p>

<p><em>[Tutorial]</em> <strong>Claire Duvallet</strong> and Mehrbod Estaki. “<a href="https://docs.qiime2.org/2018.11/tutorials/qiime2-for-experienced-microbiome-researchers/">QIIME 2 for Experienced Microbiome Researchers</a>.” <em>QIIME 2 docs.</em> (2018).</p>

<p><em>[Tutorial]</em> <strong>Claire Duvallet</strong>. “<a href="https://dev.qiime2.org/latest/tutorials/first-plugin-tutorial/">Developing a plug-in for dummies</a>.” <em>QIIME 2 developer docs.</em> (2018).</p>

<p><em>[Tutorial]</em> <strong>Claire Duvallet</strong>. “<a href="https://dev.qiime2.org/latest/tutorials/conda-tutorial/">Publishing your plugin on conda</a>.” <em>QIIME 2 developer docs.</em> (2018).</p>

<p><em>[Blog post]</em> <strong>Claire Duvallet</strong>. “<a href="https://microbe.net/2018/06/04/fuzzy-zeros-in-percentile-normalization-method-to-correct-for-batch-effects/">Fuzzy zeros in percentile normalization method to correct for batch effects</a>.” <em>microBEnet: the microbiology of the Built Environment network.</em> June 2018.</p>

<p><em>[Blog post]</em> <strong>Claire Duvallet</strong>. “<a href="http://go.nature.com/2As9meL">Beyond dysbiosis: disease-specific and shared microbiome responses to disease.</a>” <em>Nature Microbiology Community Forum.</em> December 2017.</p>

<p><em>[Dataset]</em> <strong>Claire Duvallet</strong>, Sean M. Gibbons, Thomas Gurry, Rafael A. Irizarry, and Eric J. Alm. (2017). MicrobiomeHD: the human gut microbiome in health and disease. <em>Zenodo</em>. doi: <a href="https://doi.org/10.5281/zenodo.797943">10.5281/zenodo.797943</a></p>

<h2 id="software">Software</h2>

<p><strong>Percentile normalization</strong>   <br />
Correcting batch effects in case-control microbiome studies. (<em>Gibbons et al., 2018</em>)</p>

<ul>
  <li>Python implementation: <a href="https://github.com/seangibbons/percentile_normalization">seangibbons/percentile_normalization</a> <em>(contributor)</em></li>
  <li><a href="https://qiime2.org/">QIIME 2</a> plugin: <a href="https://github.com/cduvallet/q2-perc-norm">cduvallet/q2-perc-norm</a> <em>(developer)</em></li>
</ul>

<p><strong>Distribution-based OTU calling</strong>  <br />
New implementation of Preheim <em>et al.</em>’s distribution-based OTU clustering algorithm. (<em>Preheim et al., 2013</em>; <em>Olesen et al., 2017</em>).</p>

<ul>
  <li>Python implementation: <a href="https://github.com/swo/dbotu3">swo/dbotu3</a> <em>(contributor)</em></li>
  <li><a href="https://qiime2.org/">QIIME 2</a> plugin: <a href="https://github.com/cduvallet/q2-dbotu">cduvallet/q2-dbotu</a> <em>(developer)</em></li>
</ul>

<p><strong>Amplicon sequencing pipeline</strong>    <br />
End-to-end pipeline to process 16S data.</p>

<ul>
  <li>Python: <a href="https://github.com/thomasgurry/amplicon_sequencing_pipeline">thomasgurry/amplicon_sequencing_pipeline</a> <em>(co-developer)</em></li>
</ul>

<p><strong>Match HMDB</strong>   <br />
Script to match mz’s to the HMDB database.</p>

<ul>
  <li>Python: <a href="https://github.com/cduvallet/match_hmdb">cduvallet/match_hmdb</a> <em>(developer)</em></li>
</ul>

<h2 id="other">Other</h2>

<p><em>[Blog post]</em> <strong>Claire Duvallet</strong>. “<a href="https://gradadmissions.mit.edu/blog/well-kept-secret-finding-job-post-phd">A Well-Kept Secret for Finding a Job post-PhD</a>.” <em>MIT Graduate Admissions blog</em> (2020).</p>

<p><em>[Blog post]</em> <strong>Claire Duvallet</strong>. “<a href="https://gradadmissions.mit.edu/blog/learning-engage-deep-conversations">Learning to Engage in Deep Conversations</a>.” <em>MIT Graduate Admissions blog</em> (2018).</p>

<p><em>[Resource]</em> <strong>Claire Duvallet</strong> and Monika Avello. “<a href="/files/2018-04-01-Grad-Support-Flowchart-MIT-Digital.pdf">Graduate student support resources flowchart.</a>” I and another student on <a href="http://refs.mit.edu/irefs/">iREFS</a> created this flowchart to help graduate students find support at MIT.</p>

        
      </section>

      <footer class="page__meta">
        
        




      </footer>

      

      


    </div>

    
  </article>

  
  
</div>


    </script>

    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->
<a href="/sitemap/">Sitemap</a>
<!-- end custom footer snippets -->

        

<div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
      <li><a href="https://twitter.com/dzakwanfalihh"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
    
    
    
      <li><a href="http://github.com/dzakwanfalih"><i class="fa fa-fw fa-github" aria-hidden="true"></i> GitHub</a></li>
    
    
    <li><a href="http://localhost:4000/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2021 Dzakwan Falih. Powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

      </footer>
    </div>

    <script src="http://localhost:4000/assets/js/main.min.js"></script>




  <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', '', 'auto');
  ga('send', 'pageview');
</script>






  </body>
</html>

